decred/decred/crypto/crypto.py

```diff
@@ -21,8 +21,6 @@
 from .secp256k1.curve import PrivateKey, PublicKey, curve as Curve
 
 
-KEY_SIZE = 32
-HASH_SIZE = 32
```

Comment:
These are moved to `rando.py` because they're only used there, and to avoid circular imports.